pyrasterframes/src/main/python/tests/IpythonTests.py

```diff
@@ -60,3 +63,23 @@ def test_all_nodata_tile(self):
             self.fail("test_all_nodata_tile failed with Py4JJavaError")
         except:
             self.fail("um")
+
+    def test_display_extension(self):
+        # noinspection PyUnresolvedReferences
+        import pyrasterframes.rf_ipython
+
+        self.create_layer()
+        ip = globalipapp.get_ipython()
+
+        num_rows = 2
+        row_count = 0
+
+        def counter(data, _):
+            nonlocal row_count
```

Comment:
🤔

Reply:
New in Python 3.x, so you don't have to use `global` and have the variable in the outermost scope.